tasks.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"encoding/csv"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/EFForg/starttls-backend/checker"
+	"github.com/EFForg/starttls-backend/db"
+)
+
+func runTask(name string, db db.Database) {
+	switch name {
+	case "update-stats":
+		updateStats("http://downloads.majestic.com/majestic_million.csv", db)
```

Comment:
HTTPS? I'm also a bit iffy about depending on this endpoint existing.

WDYT about having an env var pointing at its location on the machine and updating it once a week or so?